OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -972,8 +972,8 @@ public void ResolveOrder(Actor self, Order order)
 					UnReserve();
 
 				var target = Target.FromCell(self.World, cell);
-				self.SetTargetLine(target, Color.Green);
-				self.QueueActivity(order.Queued, new Fly(self, target));
+				self.QueueActivity(order.Queued, new Fly(self, target, null, Color.Green));
```

Comment:
Style nit: another case where we can do `targetLineColor: Color.Green` and skip the `null`.

Reply:
Fixed.